README.md

```diff
@@ -280,6 +280,45 @@ When the [Dev Service](#dev-service) is started, the Backstage URL is reported i
 The URL above, can be used to access the Backstage UI.
 Alternatively, the Backstage UI can be accessed from the `Backstage` card in the Dev UI: `http://localhost:8080/q/dev-ui`
 
+If you need to access [Gitea](https://about.gitea.com/) the URL is reported in the console:
+
+```
+2024-11-05 09:04:58,723 INFO  [io.qua.jgi.dep.JGitDevServicesProcessor] (build-47) Gitea HTTP URL: http://localhost:32769
+```
+
+The default credentials for the [Gitea](https://about.gitea.com/) Dev Service are: `quarkus` / `quarkus`.
+[Gitea](https://about.gitea.com/) can be accessed from the `Gitea` card in the Dev UI: `http://localhost:8080/q/dev-ui`
+
+### Container image
+
+The default container image used by the Dev Service is `quay.io/iocanel/backstage:latest` and can be changed using the following property:
+
+```properties 
+quarkus.backstage.devservices.image=<custom image>
+```
+
+The source code of the image can be found at [github.com/iocanel/backstage-docker](https://github.com/iocanel/backstage-docker).
+
+#### Custom image requirements
+
+To use a custom image, the following environment variables need to be supported: 
+
+- BACKSTAGE_TOKEN: The token used by the Backstage back end for service to service communication
+- GITHUB_TOKEN: The token used by the Backstage back end to interact with Github
```

Comment:
This environment var should be labalized as optional as by default using quarkus dev, you will use gitea

Reply:
You will still need the token, if you use the non-dev template.